src/libraries/System.Linq/src/System/Linq/OrderedEnumerable.cs

```diff
@@ -343,6 +376,9 @@ internal override int CompareAnyKeys(int index1, int index2)
         protected override void QuickSort(int[] keys, int lo, int hi) =>
             new Span<int>(keys, lo, hi - lo + 1).Sort(CompareAnyKeys);
 
+        protected override void QuickSortElements(TElement[] element, int lo, int hi) =>
```

Comment:
Similarly, I would call this something like `QuickSortInPlace` or something to that effect.